lms/templates/courseware/program_marketing.html

```diff
@@ -1,3 +1,404 @@
 <%page expression_filter="h"/>
+<%inherit file="../main.html"/>
+<%!
+from datetime import datetime
 
-## This page is intentionally left blank. You can add your own program marketing page using comprehensive theming (http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html?highlight=theming).
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangolib.markup import HTML, Text
+%>
+
+<%namespace name='static' file='../static_content.html'/>
+
+<%
+faq = program['faq']
+program_type = program['type']
+title = program['title']
+status = program['status']
+courses = program['courses']
+subtitle = program['subtitle']
+overview = program['overview']
+instructors = program['instructors']
+job_outlook_items = program['job_outlook_items']
+weeks_to_complete = program['weeks_to_complete']
+full_program_price_format = '{0:.0f}' if program['full_program_price'].is_integer() else '{0:.2f}'
+full_program_price = full_program_price_format.format(program['full_program_price'])
+endorsement = program['individual_endorsements'][0] if program['individual_endorsements'] else {}
+endorsement_quote = endorsement.get('quote', '')
+endorser = endorsement.get('endorser', {})
+endorsement_image = endorser.get('profile_image', {}).get('medium', {}).get('url', '')
+endorsement_name = endorser.get('given_name', '')
+endorsement_position = (endorser.get('position') or {}).get('title', '')
+endorsement_organization = (endorser.get('position') or {}).get('organization_name', '')
+expected_learning_items = program['expected_learning_items']
+authoring_organizations = program['authoring_organizations']
+min_hours_effort_per_week = program['min_hours_effort_per_week']
+max_hours_effort_per_week = program['max_hours_effort_per_week']
+video_url = (program.get('video', {}) or {}).get('src', '')
+banner_image = program.get('banner_image', {}).get('large', {}).get('url', '')
+description_max_length = 250
+%>
+
+<%static:css group='style-main-v2'/>
+<%static:css group='style-learner-dashboard'/>
+
+<%block name="js_extra">
+  <script src="${static.url('js/slick.min.js')}"></script>
+  <script src="${static.url('js/leanModal.js')}"></script>
+  <script src="${static.url('js/program_marketing.js')}"></script>
+</%block>
+
+<%block name="pagetitle">${program['title']}</%block>
```

Comment:
Context is different. This template has `title` variable, but it's not sent to the parent mako template.

I guess that's the reason why it's not used here.